base/docs/basedocs.jl

```diff
@@ -555,7 +555,9 @@ Short-circuiting boolean OR.
 kw"||"
 
 """
-    ccall((function_name, library) or function_pointer, returntype, (argtype1, ...), argvalue1, ...)
+    ccall((function_name, library), returntype, (argtype1, ...), argvalue1, ...)
+
```

Comment:
I think other cases of multiple signatures in the same docstring don't have a blank line between them